application/asset/js/resource-form.js

```diff
@@ -227,16 +226,25 @@
         return values;
     };
 
+    var makeDefaultValue = function (term, dataType) {
+        return makeNewValue(term, dataType)
+            .addClass('default-value')
+            // FIXME When the template is removed clicking directly on the cross, this event is skipped, so the content is removed.
```

Comment:
Instead of FIXMEs and TODOs within the code, please file an issue or call attention to an existing issue.

As for this specifically, I can't confirm an issue with the `change` event here, it seems to work as expected. Unless I'm misunderstanding the comment.

Reply:
The issue is probably outdated, I'll remove the comment.